lightning.install

```diff
@@ -513,18 +513,16 @@ function lightning_update_8004() {
  * Installs Contact Storage.
  */
 function lightning_update_8005() {
-  if (!\Drupal::moduleHandler()->moduleExists('contact_storage')) {
-    \Drupal::service('module_installer')->install(['contact_storage']);
-  }
+  // Contact and Contact Storage are now dependencies of Lightning Contact Form
```

Comment:
Let's move this comment to the function doc comment and say which version of Lightning gutted this hook. There is precedent for this in Lightning Media's install file.